widgets/FileAppender-batchsink.json

```diff
@@ -15,7 +15,10 @@
         {
           "widget-type": "textbox",
           "label": "Field separator",
-          "name": "fieldSeparator"
+          "name": "fieldSeparator",
+          "widget-attributes": {
+            "default": ","
```

Comment:
why is this change needed now?

Reply:
1. to make it explicit to end user
2. to make it similar to other plugins